doc/ocp_lock/lock_spec.ocp

```diff
@@ -759,15 +759,10 @@ Table: GET_ALGORITHMS output arguments
 |                        |        | - Byte 0 bit 1: 0x0a25, 0x0002, 0x0002            |
 |                        |        | [@{draft-irtf-cfrg-hybrid-kems}]                  |
 +------------------------+--------+---------------------------------------------------+
-| mpk_algorithms         | u32    | Indicates the size of MPKs:                       |
+| access_key_sizes       | u32    | Indicates the length of plaintext access keys:    |
 |                        |        |                                                   |
 |                        |        | - Byte 0 bit 0: 256 bits                          |
 +------------------------+--------+---------------------------------------------------+
-| access_key_algorithm   | u32    | Indicates the size of access keys:                |
```

Comment:
I decided to remove the access key algorithm. I figured we should either advertise everything (add sek and dek) or only the things that can be customized.